.codeclimate.yml

```diff
@@ -0,0 +1,15 @@
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+        javascript:
+          mass_threshold: 100
+ratings:
+  paths:
+  - src/js/**
+exclude_paths:
```

Comment:
@raquelromano-gov It's doing something. I removed those excluded paths and CC updated to say there where 156 open issues (instead of none).